STEP_Project/OUListContent.js

```diff
@@ -1,10 +1,43 @@
+let OBJ;
+if (chrome.storage != undefined)
+{
+    OBJ = chrome.storage.sync;
+}
+
 /**
  * Function that will later have Add/Remove functionality
  * @param {JS event} event - expect click event
  */
 function addRemoveButtonClick (event) 
 {
-    if (event.target.getAttribute("class")!=='bClass') return;
+    let classname = event.target.getAttribute('class')
+    if (classname !="pClass" && classname !="mClass")
+    {
+        return;
+    }
+
+    let row = event.target.parentElement;
+    var dataid = row.getAttribute("data-row-id");
+
+    if (classname=="pClass")
+    { 
+        event.target.innerHTML="-";
+        event.target.setAttribute('class','mClass'); 
+        let dataname = "name";
+        if (row.children[0].children[0] != undefined) //if there is an OU tree in the DOM
```

Comment:
- Yes, because we only call this function for the particular row that was clicked. It's _<tbody>_ child will always be there which will have the _<td>_ child. 

- What does validation mean? Send an alert that says entity added?

Reply:
We also unit test the addButtons function and make sure that it adds to storage. So we don't need to worry about checking again if clicking on the "+" buttons adds to storage.

Reply:
What I meant validations are scenarios such as child not present or dataname not present i.e. anything we find not inline with our expectation of DOM. We should fail and return right there before the plus button is changes to -.

Reply:
Validations added